lib/api/controllers/securityController.js

```diff
@@ -550,3 +564,57 @@ function resetProfiles () {
     .then(() => this.internalEngine.createOrReplace('profiles', 'default', {policies: [{roleId: 'default'}]}));
 }
 
+/**
+ *
+ * @param {Kuzzle} kuzzle
+ * @param {string.<profile|role|user>} type
+ * @param {Request} request
+ * @returns {Promise.<*>}
+ */
+function mDelete (kuzzle, type, request) {
+  const
+    action = _.upperFirst(type);
+
+  assertHasBody(request, `security:${action}`);
```

Comment:
shouldn't it be `security:delete${action}` ? (if depends on #616 as suggested below, it becomes useless)